docs/classifieds/index.md

```diff
@@ -2,14 +2,18 @@
 
 > Classifieds or Classified advertising is free of charge in the newspaper. Keep them short and to the point.
 
+[<h2>**AraCon is Coming in January**</h2>](https://aracon.one)
+[REGISTER BUILDING ORGANIZATIONS & GOVERNANCE OF THE FUTURE](https://aracon.one/registration/) 29th-30th of January 2019 in Berlin Berlin, Germany Jan 29-30.2019 The Aragon Conference will bring together people who embody and wish to shape the Aragon community. Those who are building and supporting the pseudonymous, secure, trustless future — the decentralized web.
+[Learn More](https://aracon.one/faq) |
+
 [<h2>**Senior software dev/architect at P2P Models**</h2>](https://p2pmodels.eu/jobs-senior-dev/) |
```

Comment:
Should be removed, was included in the previous issue and hasn't been re-submitted for this Issue.